contracts/interfaces/IOperators.sol

```diff
@@ -9,14 +9,14 @@ interface IOperators {
     error OperatorAlreadyRemoved();
```

Comment:
Maybe `NotAnOperator` error instead (since it could have never been an operator to begin with).